config/clusters/eks_variables.tf

```diff
@@ -30,20 +32,64 @@ variable "eks_default_worker_group_asg_max_capacity" {
 
 variable "eks_default_worker_group_additional_userdata" {
   default     = 1
-  description = "Uerdata to append to the default userdata of the default EKS worker group"
+  description = "Userdata to append to the default userdata of the default EKS worker group"
+}
+
+# Jobs on x86 Node Group.
+
+variable "eks_jobs_worker_group_name" {
+  default     = "jobs-worker-group"
+  description = "The name of the jobs EKS worker group"
+}
+
+variable "eks_jobs_worker_group_instance_type" {
+  default     = "m5.large"
+  description = "The instance type of the jobs EKS worker group"
+}
+
+variable "eks_jobs_worker_group_asg_min_capacity" {
+  default     = 1
+  description = "The Autoscaling Group size of the jobs EKS worker group"
+}
+
+variable "eks_jobs_worker_group_asg_desired_capacity" {
+  default     = 4
+  description = "The Autoscaling Group size of the jobs EKS worker group"
+}
+
+variable "eks_jobs_worker_group_asg_max_capacity" {
```

Comment:
Shouldn't we upload namings in prow.tfvars too? Ie: https://github.com/falcosecurity/test-infra/blob/master/config/clusters/prow.tfvars#L10C50-L12

Reply:
Good catch @FedeDP, thank you.

Reply:
Whoops, sorry, I misunderstood. We don't need it, we have default values right now